tsconfig.json

```diff
@@ -0,0 +1,66 @@
+{
+  "compilerOptions": {
+    /* Basic Options */
```

Comment:
should we just remove all the commented lines? seems weird to have them all in here

Reply:
i think i copied this from somewhere or it was generated. it was at least a few weeks ago so i dont remember.

i dont have a strong preference either way. its just the way it was (generated or copied) and i didnt see any value in changing it

Reply:
i think this is another place where it's a tradeoff eitherway. tsc saw value in generating the file this way for readers.

Reply:
you could argue now that the decisions are made we dont need the comments anymore. seems reasonable. i'll remove them

Reply:
btw i think i copied this from setup-node